composer.json

```diff
@@ -16,6 +16,12 @@
             "homepage": "https://github.com/sonata-project/SonataAdminBundle/contributors"
         }
     ],
+    "repositories":[
+        {
+            "type":											"vcs",
+            "url":											"https://github.com/wodka/JMSDiExtraBundle"
```

Comment:
I will remove the repositories property once the pull request in jms/di-extra-bundle is through (it's just to get it testable now)